.github/workflows/run_tests.yml

```diff
@@ -0,0 +1,35 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Run Staging Service Tests
+
+on:
+  push:
+    branches: [ develop, master ]
+  pull_request:
+    branches: [ develop, master ]
```

Comment:
unless you have a specific reason to limit testing to those branches, I would turn on tests for all branches on push and PR:

```yaml
on:
  [push, pull_request]
```